src/react-components/ui-root.js

```diff
@@ -675,100 +682,104 @@ class UIRoot extends Component {
-                srcSet="../assets/images/level_background@2x.png 2x"
-                className="audio-setup-panel__levels__icon-part"
-              />
-              <img
-                src="../assets/images/level_fill.png"
-                srcSet="../assets/images/level_fill@2x.png 2x"
-                className="audio-setup-panel__levels__icon-part"
-                style={micClip}
-              />
-              {this.state.audioTrack ? (
+          <div>
+            <div className="audio-setup-panel__title">
+              <FormattedMessage id="audio.title" />
+            </div>
+            <div className="audio-setup-panel__subtitle">
+              {(mobiledetect.mobile() || this.state.enterInVR) && (
+                <FormattedMessage id={mobiledetect.mobile() ? "audio.subtitle-mobile" : "audio.subtitle-desktop"} />
+              )}
+            </div>
+            <div className="audio-setup-panel__levels">
+              <div className="audio-setup-panel__levels__icon">
                 <img
-                  src="../assets/images/mic_level.png"
-                  srcSet="../assets/images/mic_level@2x.png 2x"
+                  src="../assets/images/level_background.png"
+                  srcSet="../assets/images/level_background@2x.png 2x"
                   className="audio-setup-panel__levels__icon-part"
                 />
-              ) : (
                 <img
-                  src="../assets/images/mic_denied.png"
-                  srcSet="../assets/images/mic_denied@2x.png 2x"
+                  src="../assets/images/level_fill.png"
+                  srcSet="../assets/images/level_fill@2x.png 2x"
                   className="audio-setup-panel__levels__icon-part"
+                  style={micClip}
                 />
-              )}
-            </div>
-            <div className="audio-setup-panel__levels__icon" onClick={this.playTestTone}>
-              <img
-                src="../assets/images/level_background.png"
-                srcSet="../assets/images/level_background@2x.png 2x"
-                className="audio-setup-panel__levels__icon-part"
-              />
-              <img
-                src="../assets/images/level_fill.png"
-                srcSet="../assets/images/level_fill@2x.png 2x"
-                className="audio-setup-panel__levels__icon-part"
-                style={speakerClip}
-              />
-              <img
-                src="../assets/images/speaker_level.png"
-                srcSet="../assets/images/speaker_level@2x.png 2x"
-                className="audio-setup-panel__levels__icon-part"
-              />
+                {this.state.audioTrack ? (
+                  <img
+                    src="../assets/images/mic_level.png"
+                    srcSet="../assets/images/mic_level@2x.png 2x"
+                    className="audio-setup-panel__levels__icon-part"
+                  />
+                ) : (
```

Comment:
It looks like something changed in this PR that causes me to see the "mic denied" image for a few frames before the audio track becomes non-null.